src/redux/books/books.js

```diff
@@ -1,28 +1,38 @@
 const ADD_BOOK = 'book-store/books/ADD_BOOK';
 const REMOVE_BOOK = 'book-store/books/REMOVE_BOOK';
 
-const bookReducer = (state = [], action) => {
+const initialState = [
```

Comment:
Great job @monicah79 🤝

- [ ] I would suggest you remove this static data since we shall use the Redux store to display books.